icons/dynamodb/dynamodb-plain.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" viewBox="0 0 128 128"><path fill="#2d72b8" d="M80.174 127.973h1.546l25.283-12.644.419-.597V13.812l-.419-.598L81.726.581h-1.568l.016 127.392m-32.904 0h-1.572L20.436 115.33l-.51-.883-.26-100.119.77-1.113L45.703.581h1.593l-.026 127.392"/><path d="M45.744.58H81.7v127.393H45.744Z" class="st2"/><path fill="#2d72b8" d="m120.244 61.416-.852-.189-12.2-1.138-.179.092-25.308-1.011H45.75l-25.308 1.01v-19.3l-.026.016.026-.056 25.308-5.865H81.7l25.308 5.865 9.306 4.946v-3.18l3.93-.429-.408-.796-12.45-8.902-.378.122-25.308-7.84H45.744L20.436 32.6V13.214L7.2 28.717v13.542l.102-.077 3.828.419v3.246L7.2 47.95v13.476l.102-.01 3.828.056v5.656l-3.302.05-.628-.06v13.48l3.93 2.114v3.282l-3.767.413-.163-.122v13.537l13.236 15.507V95.943l25.308 7.84H81.7l25.38-7.866.336.148 12.287-8.77.536-.872-3.93-.434v-3.216l-.516-.204-8.407 4.533-.306.638-25.38 5.829v.006H45.744v-.006l-25.308-5.855V68.348l25.308 1.01v.026H81.7l25.308-1.031.577.265 11.827-1.016.832-.408-3.93-.056v-5.656l3.93-.066"/><path d="M107.008 95.943v19.386l13.236-15.507v-13.45l-13.16 9.55-.076.02m0-8.187.077-.015 13.159-7.08V67.128l-13.236 1.215v19.412m.077-46.9-.077-.015v19.34l13.236 1.236V47.889l-13.16-7.034m0-8.177 13.16 9.494v-13.45l-13.236-15.508v19.387l.077.026v.05" class="st2"/></svg>
```

Comment:
I'm thinking of a way to keep the logo original but with only one color. `plain` version should have one color only. 🤔
Do you folks have any ideas of what can be done?

Reply:
```suggestion
<svg viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M80.174 127.973h1.546l25.283-12.644.419-.597V13.812l-.419-.598L81.726.581h-1.568l.016 127.392Zm-32.904 0h-1.572L20.436 115.33l-.51-.883-.26-100.119.77-1.113L45.703.581h1.593l-.026 127.392Z" fill="#2D72B8"/><path d="M45.7441.58h35.956v127.393h-35.956V.58Z" fill="#2D72B8"/><path d="m120.244 61.416-.852-.189-12.2-1.138-.179.092-25.3078-1.011h-35.955l-25.308 1.01v-19.3l-.026.016.026-.056 25.308-5.865h35.95l25.3078 5.865 9.306 4.946v-3.18l3.93-.429-.408-.796-12.45-8.902-.378.122-25.3078-7.84h-35.956L20.4362 32.6V13.214L7.2002 28.717v13.542l.102-.077 3.828.419v3.246l-3.93 2.103v13.476l.102-.01 3.828.056v5.656l-3.302.05-.628-.06v13.48l3.93 2.114v3.282l-3.767.413-.163-.122v13.537l13.236 15.507V95.943l25.308 7.84h35.956l25.3798-7.866.336.148 12.287-8.77.536-.872-3.93-.434v-3.216l-.516-.204-8.407 4.533-.306.638-25.3798 5.829v.006h-35.956v-.006l-25.308-5.855V68.348l25.308 1.01v.026h35.956l25.3078-1.031.577.265 11.827-1.016.832-.408-3.93-.056v-5.656l3.93-.066Z" fill="#2D72B8"/><path d="M107.008 95.943v19.386l13.236-15.507v-13.45l-13.16 9.55-.076.021Zm0-8.188.077-.015 13.159-7.08V67.128l-13.236 1.215v19.412Zm.077-46.9-.077-.015v19.34l13.236 1.236V47.889l-13.159-7.034Zm-.001-8.177 13.16 9.494v-13.45l-13.236-15.508v19.387l.077.026v.05" fill="#2D72B8"/></svg>
```

This could work but I would suggest not to have `plain` version. It doesn't look that good.